### DISCUSSION TITLE
Adding more map modes.

### DISCUSSION THREAD

**Discussion by mkprogz** [PiotrMachowski/lovelace-xiaomi-vacuum-map-card#224] [Q&A] [answered]
I have only two map modes by default, how to add Zones List, Rooms and Path to map modes?
![obraz](https://user-images.githubusercontent.com/32841736/143476464-8861d126-9262-4a87-bd9a-beed6670cde1.png)

**Comment by PiotrMachowski** [comment, marked as answer]
[Like that](https://github.com/PiotrMachowski/lovelace-xiaomi-vacuum-map-card/blob/master/docs/demo_config.yaml#L9)
